pkg/kubelet/cm/container_manager.go

```diff
@@ -107,7 +108,10 @@ type ContainerManager interface {
 	GetDevices(podUID, containerName string) []*podresourcesapi.ContainerDevices
```

Comment:
I know this isnÄt part of your change -- but for consistency, this really should be `devicemanager.ResourceDeviceInstances`  as it now is for `GetAllocatableDevices()` below.

Reply:
fair point.